src/components/ItaliaTheme/View/Commons/OfficeCard.jsx

```diff
@@ -103,11 +104,56 @@ const OfficeCard = ({
             })}
           </div>
         )}
+        {view_servizio && (
+          <div className="contact-info">
+            {office_fo.street && (
+              <p className="card-text">{office_fo.street}</p>
+            )}
+            {office_fo.contact_info?.map((el) =>
+              el.value_punto_contatto?.map(
+                (pdc) =>
+                  pdc.pdc_type === 'telefono' &&
+                  pdc.pdc_type !== undefined && (
+                    <div>
+                      <a
+                        href={`tel:${pdc.pdc_value}`}
+                        className="card-link pdc"
+                      >
+                        {pdc.pdc_value}
+                      </a>
+                    </div>
+                  ),
+              ),
+            )}
+
+            {office_fo.contact_info?.map((el) =>
+              el.value_punto_contatto?.map(
+                (pdc) =>
+                  pdc.pdc_type === 'email' && (
+                    <div>
+                      <a
+                        href={`mailto:${pdc.pdc_value}`}
+                        className="card-link pdc"
+                      >
+                        {pdc.pdc_value}
+                      </a>
+                    </div>
+                  ),
+              ),
+            )}
+          </div>
+        )}
         {children && <div className="card-text">{children}</div>}
       </div>
       <div className="image-container">
-        {office.preview_image?.scales?.preview && (
-          <img src={office.preview_image?.scales?.preview} alt={office.id} />
+        {office_fo?.image_scales?.preview_image[0]?.scales?.preview
+          ?.download && (
+          <img
+            src={
+              office_fo.image_scales?.preview_image[0]?.scales?.preview.download
+            }
+            alt={office.id}
+          />
```

Comment:
```suggestion
          <img
            src={
              office_fo.image_scales?.preview_image[0]?.scales?.preview.download
            }
            alt={office.id}
            loading="lazy"
            decoding="async"
          />
```

Aggiungi queste due righe per ottimizzare le performance del sito

Reply:
Done!